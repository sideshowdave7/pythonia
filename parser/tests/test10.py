assert 5, 4
assert 1 == 2
assert True

