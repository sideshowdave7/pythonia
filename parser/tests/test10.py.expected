(program (assert 5 4) (assert (comparison 1 ("==" 2))) (assert True))

